packages/runtime/test/transport/relationships.test.ts

```diff
@@ -489,7 +493,7 @@ describe("RelationshipTermination", () => {
                 to: [services2.address]
             }
         });
-        expect(result).toBeAnError(/.*/, "error.transport.messages.missingOrInactiveRelationship");
+        expect(result).toBeAnError(/.*/, "error.transport.messages.missingOrWrongRelationshipStatusOrPeerInDeletion");
```

Comment:
To many or's in one error message. We need meaningful error codes that let the user know what to do.